HTTPRequest/HTTPRequest.Unity.cs

```diff
@@ -12,71 +12,72 @@
 
 using UnityEngine;
 
-namespace Wizcorp.MageSDK.Network.Http {
-	public class HTTPRequest {
+namespace Wizcorp.MageSDK.Network.Http
+{
+	public class HttpRequest
+	{
 		private WWW request;
 		private CookieContainer cookies;
 		private Action<Exception, string> cb;
-		private Stopwatch timeoutTimer;
 
 		// Timeout setting for request
-		private long _timeout = 100 * 1000;
-		public long timeout {
-			get {
-				return _timeout;
-			}
-			set {
-				_timeout = value;
-			}
-		}
+		public long Timeout = 100 * 1000;
```

Comment:
Can't as it is supposed to be changeable by the user but has a default value. Though i agree the current implementation is a bit meh for both the dotnet and unity implementations. Make a note of it and we can improve it later.